modules/site/src/app/examples/button/button-basic-usage.component.spec.ts

```diff
@@ -1,4 +1,11 @@
-import {beforeEach, beforeEachProviders, describe, expect, it, inject,} from '@angular/core/testing';
+import {
+  beforeEach,
+  beforeEachProviders,
+  describe,
+  expect,
+  it,
+  inject,
```

Comment:
I'd prefer to figure out how to tweak the style it uses. It's really nice to make sure code is all stylistically consistent. It's been my experience that even professional developers have a hard time being consistent with each other if you don't have a build process in the middle double-checking it.